PennMobile/src/main/java/com/pennapps/labs/pennmobile/classes/GSRReservation.kt

```diff
@@ -37,4 +37,21 @@ class GSRReservation {
     @Expose
     @JvmField
     var info: Map<String, String>? = null
+
+    override fun equals(other: Any?): Boolean {
+        return other is GSRReservation && this.booking_id == other.booking_id && this.name == other.name
```

Comment:
why can't you just compare hashcode here? @meiron03 (Just curious feel free to roast if I'm wrong)